### DISCUSSION TITLE
Google Vector map features and zIndex... any way to control this?

### DISCUSSION THREAD

**Discussion by travstw** [visgl/deck.gl#6318] [Q&A]
When using a vector google map, the deck overlay renders underneath the map labels.  Is there no way to control this?  

From the Docs: 

> Shared 3D space: objects drawn by the GoogleMapsOverlay class appear inside the Google Maps scene, correctly intersecting with 3D buildings and behind the contextual labels drawn by Google Maps

**Comment by JoostKiens** [comment]
I'm a bit late to the party 🙂, but in case anyone else is interested, look at the interleaved options on `GoogleMapsOverlay`. I believe this does what you are looking for:

`new GoogleMapsOverlay({ interleaved: false })`
